source/fundamentals/connection/tls.txt

```diff
@@ -272,9 +271,7 @@ provided by Netty.
 
    The driver tests with Netty version ``{+nettyVersion+}``
```

Comment:
Tech review: Can you confirm the Netty version number used for testing.

Reply:
It's [4.1.87.Final](https://central.sonatype.com/artifact/io.netty/netty-bom/4.1.87.Final)